### PR TITLE
Fixed cloud_firestore plugin errors on latest Flutter version

### PR DESCRIPTION
**Pull request by ahmedb**
When I pull the latest Flutter + latest `cloud_firestore` plugin, I get build errors related to missing symbols. This PR updates the signatures that have changed, and removes `case` values which no longer exist

### CODE REVIEW

**Comment by ahmedb** [comment]
I signed it!

File: packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m [hunk 1]
```diff
@@ -46,8 +46,6 @@ - (FlutterError *)flutterError {
       query = [query queryWhereField:fieldName isGreaterThan:value];
     } else if ([op isEqualToString:@">="]) {
       query = [query queryWhereField:fieldName isGreaterThanOrEqualTo:value];
-    } else if ([op isEqualToString:@"array-contains"]) {
-      query = [query queryWhereField:fieldName arrayContains:value];
```
Comment (pulyaevskiy) [inline packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m]: Curious why this is going away? Is there going to be a replacement?

File: packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m [hunk 2]
```diff
@@ -114,8 +112,8 @@ - (FlutterError *)flutterError {
       @"type" : type,
       @"document" : documentChange.document.data,
       @"path" : documentChange.document.reference.path,
-      @"oldIndex" : [NSNumber numberWithInt:documentChange.oldIndex],
-      @"newIndex" : [NSNumber numberWithInt:documentChange.newIndex],
+      @"oldIndex" : [NSNumber numberWithUnsignedInteger:documentChange.oldIndex],
+      @"newIndex" : [NSNumber numberWithUnsignedInteger:documentChange.newIndex],
```
Comment (pulyaevskiy) [inline packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m]: AFAIK oldIndex and newIndex can be set to `-1` (or at least it was before). Did this change recently?

File: packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m (hunk 1 above)
Comment (ahmedb) [reply packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m]: Not sure, but I couldn't find the symbol for the `arrayContains` in latest

File: packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m (hunk 2 above)
Comment (ahmedb) [reply packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m]: Not sure, the property is declared as `NSUInteger` in latest

Comment (pulyaevskiy) [reply packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m]: @ahmedb just to double check what do you refer to as "latest"? E.g. the exact version and maybe even link to the source?

**Comment by marscj** [comment]
aaa

File: packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m (hunk 2 above)
Comment (ahmedb) [reply packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m]: Hi,

I am using flutter v0.8.2 (https://github.com/flutter/flutter/releases)

Xcode 9.4.1 on Mac OS X 10.13

And the master branch of this repository ( https://github.com/flutter/plugins )

Comment (pulyaevskiy) [reply packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m]: Thanks, I was mostly interested in versions of official pod libraries of Firebase SDK. I’m on latest 5.8.0 and arrayContains is definitely in place and working.

**Pull request closed without merging**